### PR TITLE
Added build script for sass-listen

### PR DESCRIPTION
**Pull request by atharv-phadnis**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name: **sass-listen**
Package Version: **v4.0.0**
Package URL: [sass-listen](https://github.com/sass/listen)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

**Comment by atharv-phadnis** [comment]
@Siddhesh-Ghadi please review

**Comment by Siddhesh-Ghadi** [comment]
Can you check the Travis CI logs. I am seeing some failures.

>(2\nlevels) in create_threads\'\n\nAn error occurred while installing pry-stack_explorer (0.6.1), and Bundler\ncannot continue.\n\nIn Gemfile:\n  pry-stack_explorer\n------------------listen:install_fails-------------------------------------\nhttps://github.com/sass/listen listen\nlisten  |  https://github.com/sass/listen | v4.0.0 | "Red Hat Enterprise Linux 8.5 (Ootpa)" | GitHub | Fail |  Install_Fails\n'

File: s/sass-listen/sass-listen.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 0
+fi
+
+cd $HOME_DIR/$PACKAGE_NAME
+if ! bundle exec rspec; then
+	echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails"
+	exit 0
```
Comment (Siddhesh-Ghadi) [inline s/sass-listen/sass-listen.sh:55]: Due to this the CI did not catch the failure. Please update the exit code.
```suggestion
	exit 1
```

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

File: s/sass-listen/sass-listen.sh (hunk 1 above)
Comment (atharv-phadnis) [reply s/sass-listen/sass-listen.sh]: Updated

**Review by Siddhesh-Ghadi — APPROVED** [review]
LGTM

**Pull request merged**